### ISSUE TITLE
Node.js 12 actions are deprecated

### ISSUE THREAD

**Issue by drey0143143** [terraform-docs/gh-actions#95]
I'm getting the following message when use "terraform-docs/gh-actions@v0.11.0"

```
Node.js 12 actions are deprecated. For more information see: https://github.blog/changelog/2022-09-22-github-actions-all-actions-will-begin-running-on-node16-instead-of-node12/. Please update the following actions to use Node.js 16: actions/checkout

**Comment by cocallaw** [comment]
Related issue https://github.com/terraform-docs/gh-actions/issues/92

**Comment by khos2ow** [comment]
Closing this in favor of its duplicate of #97 which is fixed in #94.
